plugins/se/client.h

```diff
@@ -42,20 +42,20 @@ struct EntityCacheInfo {
 static int32_t getDataVar(const std::string &name, procptr_t predMap) {
 	while (predMap) {
 		DataMap dataMap;
-		if (!peekProc(predMap, dataMap)) {
+		if (!proc->peek(predMap, dataMap)) {
```

Comment:
where is `proc` defined and initialized?

Reply:
It's defined in `se.cpp`, before the header is included. Definitely not ideal, in future I will probably write proper classes and pass it as `std::shared_ptr`.

Reply:
Ah yes - I see :+1: